sklearn/preprocessing/_discretization.py

```diff
@@ -163,29 +164,31 @@ def fit(self, X, y=None):
                 f"{supported_dtype + (None,)}. Got dtype={self.dtype} "
                 f" instead."
             )
-
         valid_encode = ('onehot', 'onehot-dense', 'ordinal')
         if self.encode not in valid_encode:
-            raise ValueError("Valid options for 'encode' are {}. "
-                             "Got encode={!r} instead."
-                             .format(valid_encode, self.encode))
+            raise ValueError(
```

Comment:
Please do not change unrelated things. It makes your contribution harder to review and may introduce merge conflicts to other pull requests.